README.md

```diff
@@ -1,2 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portfolio
-My portfolio that shows my educational background, projects and technical skills.
+
+> This is a mobile-first portfolio project which is built to show projects, background and the ways to get in touch with the developer.
+
+## Built With
+
+- Major languages: HTML & CSS
+- Frameworks: N/A
+- Technologies used: Git & VScode
+
+## Live Demo (if available)
+
+N/A
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- clone this repo to your local machine.
+- open it with a text editor (VScode)
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update the **Geeting Started** section by adding some information on how to clone the repo. access the cloned directory, set up the App,... You can take a look at **[this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md)** and see how to fill the section.